biohackrxiv-paper-2022/paper-20/paper.md

```diff
@@ -0,0 +1,196 @@
+
+The goal of this project was to develop an onboarding suite consisting of simple scripts, supplemented by documentation, that would help newcomers to the EGA federation in order understand in depth the main concepts, while enabling them to get involved in the development of the technology as quickly as possible.
+
+At the same time we aimed to identify existing technologies and standards
+across FEGA nodes that can be used as a reference to upcoming nodes.
+
+This biohackathon project is a result of a nordic collaboration under the umbrella of the Nordic e-Infrastructure Collaboration (NeIC)[@NEIC] where we've had three projects over the last 7 years, [Tryggve1](https://neic.no/tryggve1/)[@Tryggve1], [Tryggve2](https://neic.no/tryggve2/)[@Tryggve2] and Heilsa[@Heilsa]. For this reason the source code and scripts are shared in the same repository as the `Operator dashboard for controlling the NeIC Sensitive Data Archive` (Project 21) from BioHackathon-Europe 2022.
+
+# Results
+
+In order to achieve the project goals, our plan to focus on the main pipeline, handling the archiving of the data submitted by users to Federated EGA. Specifically, the goal is to create a number of scripts that would lead the user/developer through the process followed from submitting a file to archiving it and making it available for downloading. The scripts will shed light on the processes under the hood, including the messaging between the services, the records stored in the database as well as the tools used for encrypting and decrypting the data.
+
+The following tasks were accomplished as part of the BioHackathon:
+
+\begin{enumerate}
+\item Identify the roles that could be part of operation of an Federated EGA node, at the same time specify the needs for each role;
+\item Identify the technical requirements joining the federation;
+\item Develop a software stack containing the main services for testing purposes and document connections between the services;
+\end{enumerate}
+
+## Roles Involved in a Federated EGA Node
+
+In order to focus our work during the biohackathon we decided to first identify the roles involved in running a Federated EGA node:
+
+- **Node Representative** - aims to understand what services need to be adjust for their node so that they can run the federated EGA in their infrastructure;
```

Comment:
```suggestion
- **Node Representative** - aims to understand what services need to be adjusted for their node so that they can run the federated EGA in their infrastructure;
```